api_app/ingestors_manager/ingestors/malware_bazaar.py

```diff
@@ -0,0 +1,135 @@
+import io
+import logging
+from datetime import datetime
+from typing import Any, Iterable
+from unittest.mock import patch
+
+import pyzipper
+import requests
+
+from api_app.ingestors_manager.classes import Ingestor
+from api_app.ingestors_manager.exceptions import IngestorRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections
+
+logger = logging.getLogger(__name__)
+
+
+class MalwareBazaar(Ingestor):
+    url: str
+    hours: int
```

Comment:
Can you add as comment the same description that you added in the `Parameter` model?
Just so that if someone only reads this code, understand what is this parameter for